src/reanimated2/hook/useFrameCallback.ts

```diff
@@ -41,13 +41,16 @@ export function useFrameCallback(
   });
 
   useEffect(() => {
+    // Unregister previous callback.
+    frameCallbackRegistry.unregisterFrameCallback(ref.current.callbackId);
```

Comment:
Was unregistering here overlooked or is it not necessary? cc @tomekzaw @piaskowyk